### ISSUE TITLE
Analyzer message should not be a question

### ISSUE THREAD

**Issue by zspitz** [AdamSpeight2008/NumberSuffix_AnalyserAndCodeFix#8]
Given the following code:
```vb
Function m(m2 As ULong, mask As ULong) As Boolean
    Dim fraction As ULong = m2 And mask
    Return fraction <> 0
End Function
```
the analyzer shouldn't be asking:

> Do you want to add the type suffixe 'UL'?

Instead the analyzer should note that the comparison is against a constant Integer:

> Comparison between 'fraction' which is ULong and 0 as Integer

and the fix should be to add the type suffix.

**Comment by AdamSpeight2008** [comment]
@zspitz The analyser doesn't just check against comparison operators, but where a constant is used that could be of the wrong type.

**Comment by AdamSpeight2008** [comment]
Fixed in #10

**Comment by zspitz** [comment]
@AdamSpeight2008 

But the diagnostic message is still phrased as a question -- `... Do you want to add the type suffix '{2}' to make the Source type {0}?`.

A better phrasing of the message might be:

> Target type is {0} and source type is {1}; use type suffix '{2}' to force the source type to {0}.

Both built-in analyzers and AFAICT third-party analyzers write the message as a statement, not a question, particularly because the message might not be used in the context of the IDE, but let's say as part of the build process.
